### PR TITLE
Fix timeseries view crashing on invalid bounds in debug mode

### PR DESCRIPTION
**Pull request by Wumpf**
### What

hit this when using the `check_all_components` script

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6639?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6639?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6639)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Review by abey79 — APPROVED** [review]

File: crates/re_space_view_time_series/src/space_view_class.rs [hunk 1]
```diff
@@ -277,7 +277,14 @@ impl SpaceViewClass for TimeSeriesSpaceView {
...
         let scalar_axis =
             ViewProperty::from_archetype::<ScalarAxis>(blueprint_db, ctx.blueprint_query, view_id);
-        let y_range = scalar_axis.component_or_fallback::<Range1D>(ctx, self, state)?;
+        let mut y_range = scalar_axis.component_or_fallback::<Range1D>(ctx, self, state)?;
+        if y_range.start() >= y_range.end() {
+            // Ensure that the range is valid - egui_plot might debug_assert otherwise.
+            // `next_up_f64` should be sufficient, but empirically it's not always enough
+            // (likely we're loosing precision due to some calculations down the line)
+            *y_range.end_mut() = y_range.start() + 1.0;
+        }
```
Comment (emilk) [inline crates/re_space_view_time_series/src/space_view_class.rs:286]: Why does `component_or_fallback::<Range1D>` return an inverted range in the first place?

Comment (emilk) [reply crates/re_space_view_time_series/src/space_view_class.rs:286]: The range I see in the debugger is `(3.0000000000000004, 3.0)`. Where does these values come from?

Comment (emilk) [reply crates/re_space_view_time_series/src/space_view_class.rs:286]: Found it: 
https://github.com/rerun-io/rerun/blob/9c4df0b8abd62929915ded783e377285a716ac0c/crates/re_space_view_time_series/src/space_view_class.rs#L660-L660

Comment (emilk) [reply crates/re_space_view_time_series/src/space_view_class.rs:286]: I'll clean this up

**Pull request merged**